### PR TITLE
Longevity for 6.7

### PR DESCRIPTION
**Pull request by lgayatri**
Fixes #https://github.com/vmware/vic/issues/7626

Added Dockerfile.6.7 for new environment to provide coverage on vSphere 6.7

### CODE REVIEW

File: tests/longevity-tests/Dockerfile.6.7 [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+FROM longevity-base
+ENV GOVC_URL=10.146.13.19
+ENV TEST_URL_ARRAY=10.146.13.19
+ENV GOVC_DATASTORE=iSCSI-5TB-1
+ENV TEST_DATASTORE=iSCSI-5TB-1
+ENV BRIDGE_NETWORK=DVPortGroupVIC
+ENV PUBLIC_NETWORK=VM Network
```
Comment (mhagen-vmware) [inline tests/longevity-tests/Dockerfile.6.7]: single quote this

File: tests/longevity-tests/Dockerfile.6.7 [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+FROM longevity-base
+ENV GOVC_URL=10.146.13.19
+ENV TEST_URL_ARRAY=10.146.13.19
+ENV GOVC_DATASTORE=iSCSI-5TB-1
```
[the remark this answers is not in the document]
Comment (lgayatri) [reply tests/longevity-tests/Dockerfile.6.7]: @mhagen-vmware , ENV GOVC_URL and TEST_URL_ARRAY are pointing to VCSA. The setup is different for 6.7 and 6.5. Should "ENV GOVC_URL=10.146.13.19" be removed?

[the remark this answers is not in the document]
Comment (mhagen-vmware) [reply tests/longevity-tests/Dockerfile.6.7]: govc_url and govc_datastore are derived and set within the test based on the test_ variables, so they don't need to explicitly be duplicated here

[the remark this answers is not in the document]
Comment (lgayatri) [reply tests/longevity-tests/Dockerfile.6.7]: Ok, done

**Comment by mhagen-vmware** [comment]
with the full switch to jenkins with this PR: #7643 this configuration is now maintained in our internal repo. I have merged these changes into that now, so the next run of longevity 6.7 jenkins job will use your hardware now

**Pull request closed without merging**